docker/mongodb/source/startup-script-mongo/initialize-mongo-rs.sh

```diff
@@ -2,7 +2,7 @@
 
 echo "Executing initialize-mongo-rs.sh"
 
-mongo --eval "mongodb = ['$NODE_HOSTNAME:32001', '$NODE_HOSTNAME:32002', '$NODE_HOSTNAME:32003'], rsname = '$RS_NAME'" --shell << EOL
+mongo --eval "mongodb = ['$NODE_HOSTNAME_ONE:32001', '$NODE_HOSTNAME_TWO:32002', '$NODE_HOSTNAME_THREE:32003'], rsname = '$RS_NAME'" --shell << EOL
```

Comment:
Aroosha, instead of reading these values from the command line, could we define a configuration file and simply load it when initializing MongoDB?
If I understand this correctly, this is exactly what we do between lines 6 and 15. But having a configuration file is likely more robust and less error prone to typos and mistakes.

Reply:
I don't know how flexible you need these host names to be. But the way I was thinking is along the lines of what we have for services_config, so you would have separate configuration files (e.g. preprod, prod, dev, etc) and load the correct configuration during mongo initialization.

The configuration file would have the node names and their ports. Similar to what we have when pointing to CMSWEB services. I could be missing specific details here as well, in that case, just disconsider my comment!

Reply:
@amaltaro that's also a way to do it, but specific ports and node names can be also configured through values.yaml file. So, if we have mongodb in production, we can configure it to have different hostnames/ports/configuration compared to the preproduction through a single helm chart. I personally think that having a separate configuration file when we can configure everything through a single Helm chart adds an additional layer to the setup! (that is one of the motivations for moving to Helm charts rather than relying on simple manifest files).

But, of course, the decision lies with you and @todor-ivanov. Let me know your thoughts!